content/influxdb/cloud-dedicated/process-data/visualize/grafana.md

```diff
@@ -1,19 +1,18 @@
 ---
-title: Use Grafana to query data with SQL
-seotitle: Use Grafana to query data stored in InfluxDB Cloud Dedicated
+title: Use Grafana to query and visualize data
 list_title: Use Grafana
 description: >
   Install and run [Grafana](https://grafana.com/) to query data stored in InfluxDB.
```

Comment:
```suggestion
  Install and run [Grafana](https://grafana.com/) to query and visualize data stored in InfluxDB.
```

Reply:
Not sure which way you meant to go with these. Some are "query data" and others are "query and visualize data".